### ISSUE TITLE
Make the search details page more accessible

### ISSUE THREAD

**Issue by Captainkirkdawson** [FreeUKGen/FreeCENMigration#775]
The layout of the page is quite unfriendly to navigation by accessible tools

E0 U10 S5 C5: P20

**Comment by AlOneill** [comment]
Do not know what @Captainkirkdawson had in mind exactly but I can see a few issues:

*  the `div.container` is closed too soon so there are no side margins visible for the 'Top of Page' and 'Report a Problem' buttons
*  the "If you believe …" text is marked up as an `h5` heading which breaks the headings hierarchy (at best!)
*  in the same piece of text, the warning about the link opening in a new tab or window should be part of the link-text (so that a screen-reader user is made aware when selecting the link from a list) — the text could do with being structured differently (will think on)
*  although both tables have a `thead` element, they use `td` as column headers rather than `th`


![record-details-cen-jan-21.png](https://images.zenhubusercontent.com/5ce6b34165b1a70acc0a9124/eb81e0f6-8869-43bb-99a9-9a04b55867a1)

**Comment by Captainkirkdawson** [comment]
As always @AlOneill I had nothing specific in mind just that it did not appear to be correct

**Comment by Captainkirkdawson** [comment]
Addressed the concerns noted by @AlOneill (I hope) as well as some other minor changes. Deployed to test3
